network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/featured_blogs.html

```diff
@@ -0,0 +1,64 @@
+{% load wagtailimages_tags %}
+
+{% if page.featured_blogs.all.count > 0 %}
+{% with page.featured_blogs.all|dictsortreversed:"blog.first_published_at" as featured_blogs %}
+<div class="row">
+  <div class="col mb-4">
+    <hr class="mb-4 hr-emphasis mt-0">
+    <h4 class="h4-heading">Latest news</h4>
+  </div>
+</div>
+<div class="row mb-3">
+  {% for blog in featured_blogs %}
```

Comment:
Are we using two lines here as a way to effect an "if there are featured blogs, use the first one"?